doc/tutorial/pairwise_alignment/pa_assignment_3_solution.cpp

```diff
@@ -27,10 +27,12 @@ int main()
     }
 
     // Configure the alignment kernel.
-    auto config = align_cfg::mode{global_alignment} |
-                  align_cfg::scoring{aminoacid_scoring_scheme{aminoacid_similarity_matrix::BLOSUM62}} |
-                  align_cfg::aligned_ends{free_ends_second};
+    auto config = seqan3::align_cfg::mode{seqan3::global_alignment} |
+                  seqan3::align_cfg::scoring{seqan3::aminoacid_scoring_scheme{
+                      seqan3::aminoacid_similarity_matrix::BLOSUM62
+                  }} |
```

Comment:
Can you move these to the end of the previous line because the matching `{}` are anyway not in the same column?